pgx_async/src/pgx_async.mli

```diff
@@ -1,23 +1,14 @@
 (** Async based Postgres client based on Pgx. *)
 open Async_kernel
 
-include Pgx.S with type 'a Io.t = 'a Deferred.t
+include
+  Pgx.S
+    with type 'a Io.t = 'a Deferred.t
+     and type Io.ssl_config = Conduit_async.Ssl.config
 
 (* for testing purposes *)
 module Thread : Pgx.Io with type 'a t = 'a Deferred.t
 
-val with_conn
-  :  ?host:string
-  -> ?port:int
-  -> ?user:string
-  -> ?password:string
-  -> ?database:string
-  -> ?unix_domain_socket_dir:string
-  -> ?verbose:int
-  -> ?max_message_length:int
-  -> (t -> 'a Deferred.t)
-  -> 'a Deferred.t
```

Comment:
This is unnecessary since it's exactly the same interface that we import above in `include Pgx.S`